addons/web/static/src/views/pivot/pivot_view.scss

```diff
@@ -1,96 +1,29 @@
 .o_pivot {
-    .o_pivot_cell_value {
-        font-size: 1em;
-        .o_comparison {
-            font-size: 0.61em;
-        }
-        .o_variation {
-            &.o_positive {
-                color: green;
-            }
-            &.o_negative {
-                color: red;
-            }
-        }
-    }
 
     table {
-        th > .dropdown > .dropdown-toggle {
-            background: none!important;
-            border: none!important;
-            padding: 0!important;
-        }
-        background-color: $o-view-background-color;
-        width: auto;  // bootstrap override
+        background-color: $table-bg;
 
         // Inform the user that he may click on a cell to be redirected to a list view of the
         // items corresponding to the clicked cell
-        &.o_enable_linking {
-            .o_pivot_cell_value:not(.o_empty):hover {
-                color: $o-brand-primary;
-                cursor: pointer;
-
-            }
-        }
-
-        thead th:not(.o_pivot_header_cell_closed):not(.o_pivot_header_cell_opened):not(.o_pivot_header_cell) {
-            text-align: center;
+        &.o_enable_linking .o_pivot_cell_value:not(.o_empty) {
+            @include o-hover-text-color($body-color, $o-brand-primary);
```

Comment:
use `$o-main-link-color` rather than `$o-brand-primary`.
Add an empty line after the mixin.